openquake/server/static/js/engine.js

```diff
@@ -153,29 +155,54 @@
                 this.can_be_rendered = true;
             },
 
+            show_modal_confirm_remove_calculation: function(e) {
+                e.preventDefault();
+                var calc_id = $(e.target).attr('data-calc-id');
+                
+                var show_or_back = (function(e) {
+                    this.conf_show = $('#confirmDialog' + calc_id).show();
+                    this.back_conf_show = $('.back_confirmDialog' + calc_id).show();
+                    closeTimer();
+                })();
+            },
+
+            hide_modal_confirm_remove_calculation: function(e) {
+                e.preventDefault();
+                var calc_id = $(e.target).attr('data-calc-id');
+                
+                var hide_or_back = (function(e) {
+                    this.conf_hide = $('#confirmDialog' + calc_id).hide();
+                    this.back_conf_hide = $('.back_confirmDialog' + calc_id).hide();
+                    setTimer();
+                })();
+            },
+
+
             remove_calculation: function(e) {
                 e.preventDefault();
                 var calc_id = $(e.target).attr('data-calc-id');
+                var calc_desc = $(e.target).attr('data-calc-desc');
                 var view = this;
                 diaerror.show(false, "Removing calculation " + calc_id, "...");
                 $.post(gem_oq_server_url + "/v1/calc/" + calc_id + "/remove"
                      ).success(
                          function(data, textStatus, jqXHR)
                          {
-                             diaerror.show(false, "Removing calculation " + calc_id, "Calculation " + calc_id + " removed.");
+                             diaerror.show(false, "Removed calculation ", "Calculation:<br><b>(" + calc_id + ") " + calc_desc + "</b> removed.");
                              view.calculations.remove([view.calculations.get(calc_id)]);
                          }
                      ).error(
                          function(jqXHR, textStatus, errorThrown)
                          {
                              if (jqXHR.status == 404) {
-                                 diaerror.show(false, "Removing calculation " + calc_id, "Failed: calculation " + calc_id + " not found.");
+                                 diaerror.show(false, "Removing calculation ", "Failed calculation:<br><b>(" + calc_id + ") " + calc_desc + "</b> not found.");
```

Comment:
`s/"Failed calculation:<br><b>/"Failed. Calculation:<br><b>/g`
otherwise it looks like that it's a "Failed calculation" not that just the removal command failed.